automation/pages/basePage.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+class basePage(ABC):
+    def __init__(self, driver):
+        self.driver = driver
+        self.baseURL = "http://127.0.0.1:8080"
+
+    def open(self,URL):
+        self.driver.get(self.baseURL + URL)
+        WebDriverWait(self.driver, 10).until(EC.title_contains("Which One"))
+        assert self.driver.title == "Which One"
```

Comment:
I think the assert is redundant because your expected condition is the same and will fail if the previous line times out.

Is that right @smangan360 ?